### ISSUE TITLE
master crash

### ISSUE THREAD

**Issue by wzrdtales** [seaweedfs/seaweedfs#4273]
```

I0306 04:34:58.843395 master_server.go:323 executing: ec.encode [-collection=backups -fullPercent=95 -quietFor=1h]
collect volumes quiet for: 3600 seconds
ec encode volumes: [820 827 828 821 823 838 835 796 797 816 798 834 833 836 822 819 817 825 824 830 829 826 837 818]
panic: runtime error: index out of range [0] with length 0

goroutine 185 [running]:
github.com/seaweedfs/seaweedfs/weed/shell.doEcEncode(0xc000c3a3a0, {0xc000c240bf, 0x11}, 0x16?, 0x0?)
        /home/ubuntu/seaweedfs/weed/shell/command_ec_encode.go:115 +0x4af
github.com/seaweedfs/seaweedfs/weed/shell.(*commandEcEncode).Do(0xc001f999b0?, {0xc001f999b0, 0x3, 0x3}, 0xc00204ff10?, {0x2?, 0x2?})
        /home/ubuntu/seaweedfs/weed/shell/command_ec_encode.go:87 +0x346
github.com/seaweedfs/seaweedfs/weed/server.processEachCmd(0xc000c240a7?, {0xc000c240a7?, 0x23f5d20?}, 0x1?)
        /home/ubuntu/seaweedfs/weed/server/master_server.go:324 +0x2b5
github.com/seaweedfs/seaweedfs/weed/server.(*MasterServer).startAdminScripts.func1()
        /home/ubuntu/seaweedfs/weed/server/master_server.go:302 +0x1d6
created by github.com/seaweedfs/seaweedfs/weed/server.(*MasterServer).startAdminScripts
        /home/ubuntu/seaweedfs/weed/server/master_server.go:292 +0x5ac

```

version running was either 3.41 or 3.43 not sure anymore

**Comment by chrislusf** [comment]
This should have been fixed in 3.43 with https://github.com/seaweedfs/seaweedfs/commit/31bb91583f1523c635d691de2c7941a35ac26fd6

**Comment by wzrdtales** [comment]
ok we're running the same demo env on 3.43 since yesterday effectively. will open a new issue if that crash persists
